caos.ansible_roles/roles/dockerized_agent/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Create folder for docker-compose
+  file:
+    path: "{{ dockerfile_directory }}"
+    state: directory
+    mode: '0755'
+
+- name: Copy Dockerfile from role template
+  template:
+    src: docker-compose.yml.j2
+    dest: "{{ dockerfile_directory }}/{{ dockerfile_name }}"
+    mode: 0644
+  when: custom_dockerfile == ""
+
+- name: Copy Dockerfile from relative playbook file
+  template:
+    src: "{{ custom_dockerfile }}"
```

Comment:
small suggestion. Would something like this work?

```yaml
- name: Copy Dockerfile from relative playbook file
  template:
    src: "{{ custom_dockerfile | default('docker-compose.yml.j2') }}"
```

Reply:
Nice suggestion! Let's try it ⚙️